src/sbt-test/site/can-run-generator-twice/build.sbt

```diff
@@ -6,17 +6,18 @@ val Site1 = config("site1")
 val Site2 = config("site2")
 
 // Apply the default Paradox settings to the `Site1` config
+ParadoxPlugin.paradoxSettings(Site1)
 ParadoxSitePlugin.paradoxSettings(Site1)
```

Comment:
This is to keep the same usage as one would do when using sbt-site with multiple configurations.